c/tskit/tables.c

```diff
@@ -7957,6 +7957,203 @@ tsk_table_collection_subset(
     return ret;
 }
 
+int TSK_WARN_UNUSED
+tsk_table_collection_union(tsk_table_collection_t *self, tsk_table_collection_t *other,
+    tsk_id_t *nodes, tsk_size_t num_nodes)
+{
+    int ret = 0;
+    tsk_id_t k, i, new_ind, new_pop, new_parent, new_child;
+    // tsk_id_t new_source, new_dest;
```

Comment:
should remove the commented-out stuff

Reply:
I kept this there for when we allow migrations.

Reply:
Once sort starts sorting migrations, we can just uncomment this line and the lines in the migration section below. Still do you think it's better to get rid of it?

Reply:
I do. If at that (undefined) point we're still motivated to go back and put this in, it'll be easy to do again - it's just following the pattern elsewhere in this function.